pkg/ccl/changefeedccl/fetch_table_bytes.go

```diff
@@ -82,7 +100,7 @@ func fetchSizes(
 		return 0, err
 	}
 	if len(resp.Errors) > 0 {
-		return 0, errors.Newf("errors fetching span stats: %v", resp.Errors)
+		return 0, errors.Newf("errors fetching span stats: %+v", safeErrorsResponse(resp.Errors))
```

Comment:
ok but how can we be sure this IS actually safe to unredact? not sure tbh. cc @rharding6373